omnibus-arbitrum-sepolia.toml

```diff
@@ -278,91 +308,3 @@ args = [
     18,
     "<%= imports.system.contracts.USDRouter.address %>",
 ]
-
-[provision.RewardsDistributor_SpartanCouncilPool_ARB_rewards_for_ARB_lp]
```

Comment:
@kaleb-keny better comment on code itself so we can have proper reply threads.

These configs were extracted into separate files as we were getting way to much of them at this point. This does not create any new transactions.